app/views/participants/validations/complete_cip.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for :title, "Registration complete" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_panel title: "Registration complete", body: nil, classes: "govuk-!-margin-bottom-8" %>
+    <h2 class="govuk-heading-m">What happens next</h2>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_panel title: title, body: nil, classes: "govuk-!-margin-bottom-8" %>
+    <h2 class="govuk-headming-m">What happens next</h2>
+    <p class="govuk-body">We may need to contact you for more information to complete your registration.</p>
+    <% if @participant.ect? %>
+      <p class="govuk-body">Your training materials will be available in August. We’ll email you with a link to access them.</p>
+    <% else %>
+      <p class="govuk-body">If you need access to training materials, we’ll email you a link in the next 48 hours.</p>
+    <% end %>
+  </div>
+</div>
```

Comment:
This view looks off. The top two divs are not closed, but also I reckon they are not needed. I think we can just axe lines 3-6. 

And then the h2 on line 10 (what happens next?) needs to be h1 for accessibility reasons. I think.

Reply:
Ohh good spot, something went wrong there